### ISSUE TITLE
Is there any way to ignore transparent background in png?

### ISSUE THREAD

**Issue by arsenanai** [jankovicsandras/imagetracerjs#39]
Hi, I'm trying to convert signature to vector image. I have png image with transparent background, but when I transform image to SVG, it fills out whole backgound with colored path. Is there any way to avoid that.
Used options:
```
            //detailed
            pathomit:0, 
            roundcoords:2, 
            ltres:0.5, 
            qtres:0.5,
            //curvy
            linefilter: true,
            rightangleenhance:false,
            //grayscale
            colorsampling:0, 
            colorquantcycles:1, 
            numberofcolors:2,
```
Another question is: Can you advice me best option to make curved, smooth and noisyless options, thanks

Images attached here:

[Archive.zip](https://github.com/jankovicsandras/imagetracerjs/files/4592116/Archive.zip)

**Comment by r-zammit** [comment]
@arsenanai were you able to find the answer to making smooth, noisyless curves? Mine look like this
<img width="165" alt="Screen Shot 2020-05-08 at 2 17 30 PM" src="https://user-images.githubusercontent.com/12617956/81442104-f6a27d80-9140-11ea-804a-02d89a7e9650.png">

**Comment by jankovicsandras** [comment]
Hi,

@arsenanai : this can be related to Issue #31  , I'm working on it.
@r-zammit : we will discuss this question in Issue #40

**Comment by jankovicsandras** [comment]
Hi,

@arsenanai : This should be fixed in version 1.2.6, please test.
@r-zammit : I close this Issue, we will continue in Issue #40
